packages/react-components/src/useGrpcClient.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import { Network } from '@concordium/wallet-connectors';
+import { ConcordiumGRPCClient } from '@concordium/web-sdk';
+import { GrpcWebFetchTransport } from '@protobuf-ts/grpcweb-transport';
+
+export interface GrpcClient {
+    grpcClient: ConcordiumGRPCClient | undefined;
+}
+
+export function useGrpcClient({ grpcOpts }: Network): GrpcClient {
```

Comment:
I did it this way for better forward compatibility if new fields were to be added, like an error. But I concluded that it will never fail and don't know what other fields could be. So I suppose we can migrate if something needs to be added at some point.